### PR TITLE
Install newer bundler

### PR DESCRIPTION
**Pull request by jaym**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 language: ruby
-cache: bundler
 
 sudo: false
 # Early warning system to catch if Rubygems breaks something
-before_install:
+before_install: gem install bundler -v 1.10.5
   # the new 2.5.0 rubygems is busted with 1.7.6 bundler, the
   # new 1.10.6 bundler is busted with our rake tasks.  the
   # 2.4.5 rubygems with 1.7.6 bundler in travis currently works.
```
Comment (lamont-granquist) [inline .travis.yml]: comment probably needs to be updated.  what about bumping/pinning rubygems?

**Comment by danielsdeleo** [comment]
:+1:

**Comment by danielsdeleo** [comment]
Though travis didn't run at all on this PR? Is that a travis problem or a `.travis.yml` problem?

**Comment by jaym** [comment]
Travis ran, you just need to look at the previous commit. I told it to skip ci for the update to the comment

**Comment by danielsdeleo** [comment]
Ok, gotcha. I was concerned the last commit might've made travis barf for some reason

**Comment by jaym** [comment]
@jkeiser fixed this

**Pull request closed without merging**